osf/models/analytics.py

```diff
@@ -63,6 +65,26 @@ class PageCounter(BaseModel):
     total = models.PositiveIntegerField(default=0)
     unique = models.PositiveIntegerField(default=0)
 
+    DOWNLOAD_ALL_VERSIONS_ID_PATTERN = r'^download:[^:]*:{1}[^:]*$'
+
+    @staticmethod
+    def get_all_downloads_on_date(date):
+        """
+        Queries the total number of downloads on a date
+        :param str date: must be formatted the same as a page counter key so 'yyyy/mm/dd'
+        :return: long sum:
+        """
+
+        # Get all PageCounters with data for the date made for all versions downloads,
+        # regex insures one colon so all versions are queried.
+        page_counters = PageCounter.objects.filter(date__has_key=date, _id__regex=PageCounter.DOWNLOAD_ALL_VERSIONS_ID_PATTERN)
```

Comment:
Seems like this should be a `classmethod` so that you don't have to hardcode `PageCounter`.

```python
page_counters = cls.objects.filter(...)
```